README.md

```diff
@@ -20,26 +23,49 @@ require('base16-colorscheme').setup({
 
 *Note: If you don't see colours, try adding `vim.opt.termguicolors = true` to your init.lua*
 
-# Advanced Usage
+## Advanced Usage
+
+To disable highlights for supported plugin(s), call the `with_config` function **before** setting the colorscheme:
 
 ```lua
--- To disable highlights for supported plugin(s), call the `with_config` function **before** setting the colorscheme.
--- These are the defaults.
+-- Call `with_config` **before** setting a colorscheme if you want to tweak anything
 require('base16-colorscheme').with_config({
+    -- These are the defaults.
     telescope = true,
+    telescope_borders = false,
     indentblankline = true,
     notify = true,
     ts_rainbow = true,
     cmp = true,
     illuminate = true,
+    lsp_semantic = true,
+    mini_completion = true,
     dapui = true,
 })
 
--- You can get the base16 colors **after** setting the colorscheme by name (base01, base02, etc.)
+-- `setup` should be called **after** `with_config`
+require('base16-colorscheme').setup('schemer-dark')
+
+-- or use `colorscheme` instead
+vim.cmd('colorscheme base16-catppuccin-mocha')
+```
+
+**After** you have called `setup` or set `colorscheme`, you can access the theme's base16 colors (base01, base02, etc.):
```

Comment:
I'd rather not have this here, loading he colorscheme should be pretty early if they wrote their config well, doing this before the colorscheme is setup is a symptom of a different issue.